### DISCUSSION TITLE
Add support for Vite

### DISCUSSION THREAD

**Discussion by JoaoVanzuita** [sushinpv/react-secure-storage#6] [Ideas]
Hello, I was looking for a solution to save data securely in the browser and I really liked this lib, but unfortunately I use Vite in my app and I couldn't use it, are there any plans to add support for Vite?

I think it would be possible to check if Vite was used using " typeof import.meta.env.VITE_SECURE_LOCAL_STORAGE_HASH_KEY === 'undefined' "

**Comment by sushinpv** [comment]
Hi @JoaoVanzuita,I was planned to move on to Vue.js next, but I don't yet have a clear schedule for doing so. Nevertheless, I anticipate being able to provide an update in the next few days. You are invited to make a contribution at the same time.

You can create a patch for the library
